repos/system_upgrade/el7toel8/libraries/dnfplugin.py

```diff
@@ -39,9 +39,9 @@ def build_plugin_data(target_repoids, debug, test, tasks):
     data = {
         'pkgs_info': {
             'local_rpms': [os.path.join('/installroot', pkg.lstrip('/')) for pkg in tasks.local_rpms],
-            'to_install': [pkg for pkg in tasks.to_install],
-            'to_remove': [pkg for pkg in tasks.to_remove],
-            'to_upgrade': [pkg for pkg in tasks.to_upgrade]
+            'to_install': tasks.to_install[:],
+            'to_remove': tasks.to_remove[:],
+            'to_upgrade': tasks.to_upgrade[:]
```

Comment:
regarding the context of the file, it seems there is no need to copy the list as the final output is generated without any steps in the middle and it doesn't seem that lists are changed in any way. So I guess it could be ok just assign the objects without copying. But it is minor thing.

Reply:
Unresolving as I see absolutely no point in copying lists here

Reply:
I'll fix it on next commit.